.github/workflows/rust.yml

```diff
@@ -67,6 +67,8 @@ jobs:
           tool: cargo-fuzz
       - name: Fuzz (nightly only)
         if: ${{ matrix.rust == 'nightly' }}
+        env:
+          RUSTFLAGS: "-C target-feature=-crt-static"
         run: |
           cargo +nightly fuzz run init -- -max_total_time=900
           cargo +nightly fuzz run calibrate -- -max_total_time=1800
```

Comment:
For musl targets the `cargo fuzz` fails:

- Issue: https://github.com/rust-fuzz/cargo-fuzz/issues/397
- See workflow run: https://github.com/eupn/bno055/actions/runs/19533061607/job/55920473048